neurodsp/timefrequency/hilbert.py

```diff
@@ -167,10 +168,10 @@ def freq_by_time(sig, fs, f_range=None, hilbert_increase_n=False,
     f_range : tuple of float or None, optional default: None
         Filter range, in Hz, as (low, high). If None, no filtering is applied.
     hilbert_increase_n : bool, optional, default: False
-        If True, zero pad the signal to length the next power of 2 when doing the Hilbert transform.
+        If True, zero pad the signal to length the next power of 2 for the Hilbert transform.
```

Comment:
This reads a bit strange. Maybe "If True, zero pad the signal's length to the next power..."